app/services/sqs/publish_hearing.rb

```diff
@@ -52,6 +52,7 @@ def message
         caseCreationDate: shared_time.to_date.strftime("%Y-%m-%d"),
         cjsLocation: cjs_location,
         docLanguage: "EN",
+        proceedingsConcluded: defendant.dig(:proceedingsConcluded),
```

Comment:
As we're not going into nested hashes, would ` &.` be preferable over `.dig` ?

Reply:
`&.` is an operator that works like `try`, https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby, so it's not particular to hashes. `dig` was designed to fetch values in nested hashes, but can also be used as a synonym to `[]`. I do agree with you that `[]` might be better, in the sense that it's more intention revealing.

I'd suggest staying away from `&.` as much as possible, because it often violates the principle of least knowledge that is best practice in object-oriented design (aka "law of Demeter"), and the "fail fast" philosophy (https://martinfowler.com/ieeeSoftware/failFast.pdf).